docs/sources/administration/plugin-management/index.md

```diff
@@ -17,7 +17,7 @@ weight: 600
 
 Besides the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience with _plugins_.
 
-You can [install]({{< relref "#install-a-plugin" >}}) one of the plugins built by the Grafana community, or [build one yourself]({{< relref "../../developers/plugins/" >}}).
+You can [install]({{< relref "#install-a-plugin" >}}) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools/).
```

Comment:
nit: the `relref` shortcode is not needed for in-page anchors, it's just extra typing. Also, this is the only URL with a trailing slash. It resolves fine either way.

```suggestion
You can [install](#install-a-plugin) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools).
```